package.json

```diff
@@ -3,7 +3,7 @@
   "publisher": "apidev",
   "displayName": "Azure API Center",
   "description": "Build, discover, and consume APIs.",
-  "version": "1.2.0",
+  "version": "1.4.0",
```

Comment:
do we still need follow the even number pattern?

Reply:
set to 1.3.0